contracts/stake/StakedTokenV3.sol

```diff
@@ -161,15 +166,36 @@ contract StakedTokenV3 is StakedTokenV2, IStakedTokenV3, RoleManager {
     _maxSlashablePercentage = maxSlashablePercentage;
   }
 
+  function setEmergencyShutdown(bool emergencyShutdown) external override onlyCooldownAdmin {
+    _emergencyShutdown = emergencyShutdown;
```

Comment:
does it make sense to check that the current _emergencyShutdown status != the input emergencyShutdown

As is, you could have a EmergencyShutdownChanged event from true to true. A lot would have to go wrong in the gov process to end up with that, but could be added to avoid events that don't make sense.

Reply:
Don't think so. We allow writing the same value to state for other configurations in pool etc, so think it is fine to keep the consistency here to also allow "rewriting" the same value.